src/keria/app/ipexing.py

```diff
@@ -101,7 +101,7 @@ def sendAdmit(agent, hab, ked, sigs, rec):
         agent.exchanges.append(dict(said=serder.said, pre=hab.pre, rec=rec, topic='credential'))
         agent.admits.append(dict(said=ked['d'], pre=hab.pre))
 
-        return agent.monitor.submit(serder.pre, longrunning.OpTypes.exchange, metadata=dict(said=serder.said))
+        return agent.monitor.submit(serder.said, longrunning.OpTypes.exchange, metadata=dict(said=serder.said))
```

Comment:
If this were an ipexing PR i'd vote to change it now. But since it's a oid PR i vote to clean it up in a follow-on PR.

Reply:
No problem cleaning it up now, it's more so that it's almost like a breaking change to an API so I'm just worried I might break some people without them realising